apiserversdk/proxy.go

```diff
@@ -46,3 +56,37 @@ func WithFieldSelector(handler http.Handler, selectors ...string) http.Handler {
 		handler.ServeHTTP(w, r)
 	})
 }
+
+func requireKuberayService(baseHandler http.Handler, config *rest.Config) (http.Handler, error) {
```

Comment:
```suggestion
func requireKuberayService(baseHandler http.Handler, client *kubernetes.Clientset) (http.Handler, error) {
```

We can safely use `kubernetes.NewForConfigOrDie` to construct the client after `proxy.Transport` without handling the error because the error has been handled when we construct the `proxy.Transport`.

Reply:
Good idea. Thank you